tools/src/output.rs

```diff
@@ -271,7 +271,6 @@ pub fn generate_footer(
     }
 
     let scripts = [
-        "dxr.js",
```

Comment:
Same here, did you test these changes? `dxr.js` was originally from dxr, though maybe we should rename it to `search.js` or something? It was heavily changed.